### ISSUE TITLE
Invalid line ending with yarn on Unix system

### ISSUE THREAD

**Issue by arthurtemple** [vendure-ecommerce/ngx-translate-extract#7]
Hello,

Thanks for your involvement in keeping this project alive!

There seems to be an issue when using your fork with yarn on a Unix system.

#### Steps to reproduce
```
yarn global add @vendure/ngx-translate-extract
ngx-translate-extract
```

This gives following output: 
```
/usr/bin/env: ‘node\r’: No such file or directory
```
(Using `@bartholomej/ngx-translate-extract` instead displays help, which is expected behavior)

#### Reproduced on
- Ubuntu 20.04
- Ubuntu 22.04
- MacOS Ventura 13.3.1

#### Side note
Using `dos2unix` or `sed -i $'s/\r$//'` on bin/cli.js fixes the issue locally.

**Comment by michaelbromley** [comment]
I just published a new version, v8.2.0 - let me know if the problem is resolved in that version.

**Comment by arthurtemple** [comment]
Problem solved, thanks a lot!
